invenio/legacy/inveniocfg.py

```diff
@@ -832,9 +832,9 @@ def _detect_ip_address():
     """
     try:
         s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        s.connect(('invenio-software.org', 0))
```

Comment:
I may be missing something here. What is the relationship between `invenio-software.org` being up and someone creating the apache configuration for their website (it doesn't have to be the demosite).